package/src/contexts/messageInputContext/MessageInputContext.tsx

```diff
@@ -830,15 +785,15 @@ export const MessageInputProvider = ({
       setSelectedImages([]);
     }
 
+    // setText('');
```

Comment:
Don't forget to remove these

Reply:
I have removed it in my attachment manager PR but will remove here as well.

Reply:
No need, these were just reminders. All good 👍